src/coprocessor/dag/rpn_expr/function.rs

```diff
@@ -18,7 +18,12 @@
 //! function name) that generates an `RpnFnMeta` struct.
 //!
 //! If you needs `EvalContext` or the raw `RpnFnCallPayload`, just put it ahead of the function
-//! parameters, and add `ctx` or `payload` argument to the attribute. For example:
+//! parameters, and add `ctx` or `payload` argument to the attribute.
+//! It can only be `#[rpn_fn]` or `#[rpn_fn(ctx)]` or `#[rpn_fn(ctx,payload)]` or `#[rpn_fn(payload,ctx)]`,
```

Comment:
`#[rpn_fn(payload)]` is missing.
I think we shouldn't list all permutations of them. It looks too redundant.